code/modules/organs/organ_internal.dm

```diff
@@ -190,7 +190,7 @@ obj/item/organ/vaurca/neuralsocket/process()
 			owner << "\blue Your mind expands, and your thoughts join the unity of the Hivenet."
 	..()
 
-/obj/item/organ/vaurca/neuralsocket/replaced(var/mob/living/carbon/human/target)
+/obj/item/organ/vaurca/neuralsocket/replaced()
```

Comment:
Not sure if smart.

Either call `..()` **before** the hivenet is added (if memory serves, replaced() is called even if the transplanting fails), and check `if (owner)`, or leave it with the target argument.